### PR TITLE
Fixes "case-insensitive"-Rule in robots.txt

### PR DESCRIPTION
**Pull request by Enaah**
Base on the Issue #8 (_Robots.txt fields should be case-insensitive_) I applied the necessary changes by simple writing all _<field>_-elements case-insensitive.
Another option could be using the `strtolower()`-Function on the   _<field>_-elements but this seems unclean.

The Issue was opend, based on the RFC-standard by google.
The important part ist:
"The <field> element is case-insensitive. The <value> element may be case-sensitive, depending on the <field> element."
(Source: https://developers.google.com/search/reference/robots_txt?hl=en)

---
I ran PHP-Unit (with the `express-server` running and all Test passed.
But an Error was thrown, saying _No code coverage driver is available_.
It has to do, that I do no have debug installed, hope this is no Problem.

If anything else is needed to accept the PR, please let me know.

Thanks @freekmurze for sharing the as a good start on open source on Twitter!

### CODE REVIEW

File: src/RobotsTxt.php [hunk 1]
```diff
@@ -104,17 +104,17 @@ protected function isCommentLine(string $line): bool
 
     protected function isUserAgentLine(string $line): bool
     {
-        return strpos(trim($line), 'User-agent') === 0;
+        return strpos(trim($line), 'user-agent') === 0;
```
Comment (freekmurze) [inline src/RobotsTxt.php]: Shouldn't we use `strtolower` on `$line` to make sure `$line` itself is lowercased?

Comment (Enaah) [reply src/RobotsTxt.php]: Yes, definitely.

And another one here, correct?
Line `115` - `src/RobotsTxt.php`
```php
 protected function parseDisallow(string $line): string
    {
        return trim(str_replace('disallow', '', trim($line)), ': ');
    }
```
to
```php
 protected function parseDisallow(string $line): string
    {
        return trim(str_replace('disallow', '', trim(strtolower($line))), ': ');
    }
```
If nothing else should be changed, I will do push the changes.

File: tests/data/robots.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # robotstxt.org/
 
-User-agent: *
```
Comment (freekmurze) [inline tests/data/robots.txt]: Maybe also revert all changes to tests. If the change to the package are correct, all tests should still pass without any changes.

**Comment by Enaah** [comment]
The changes are pushed.
Hope everything is okay.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**